src/otx/engine/engine.py

```diff
@@ -327,6 +328,61 @@ def export(self, *args, **kwargs) -> None:
         """Export the trained model to OpenVINO Intermediate Representation (IR) or ONNX formats."""
         raise NotImplementedError
 
+    def explain(
+        self,
+        checkpoint: str | Path | None = None,
+        datamodule: EVAL_DATALOADERS | OTXDataModule | None = None,
+        explain_config: ExplainConfig | None = None,
+        **kwargs,
+    ) -> list | None:
+        """Run XAI using the specified model and data.
+
+        Args:
+            checkpoint (str | Path | None, optional): The path to the checkpoint file to load the model from.
+            datamodule (EVAL_DATALOADERS | OTXDataModule | None, optional): The data module to use for predictions.
+            explain_config (ExplainConfig | None, optional): Config used to handle saliency maps.
+            **kwargs: Additional keyword arguments for pl.Trainer configuration.
+
+        Returns:
+            list: Saliency maps.
+
+        Example:
+            >>> engine.explain(
+            ...     datamodule=OTXDataModule(),
+            ...     checkpoint=<checkpoint/path>,
+            ...     explain_config=ExplainConfig(),
+            ... )
+        """
+        from pathlib import Path
```

Comment:
Why do we need to import Path here? We already have this import in TYPE_CHECKING section. It can just be moved out of it to enable the actual usage of the package

Reply:
Right, updated